python/tk_desktop/setup_project.py

```diff
@@ -62,10 +69,57 @@ def do_setup(self):
                                        "and delete all pipeline configurations for your project.")
             ret = error_dialog.exec_()
```

Comment:
ret is assigned at t couple of places but I don't see it being returned anywhere? Dead code? Might be worth removing it.